substrate/frame/revive/src/weights.rs

```diff
@@ -1223,6 +1231,13 @@ impl WeightInfo for () {
 		// Minimum execution time: 4_361_000 picoseconds.
 		Weight::from_parts(4_577_000, 0)
 	}
+	fn seal_balance_of() -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `140`
+		//  Estimated: `0`
+		// Minimum execution time: 4_361_000 picoseconds.
+		Weight::from_parts(4_577_000, 0)
```

Comment:
Hmm I would expect some storage access here.

Reply:
When I run the benchmark locally I see some proof size, I'll try again after #5684